chassis.go

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) 2020 eBay Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package goovn
+
+import (
+	"github.com/ebay/libovsdb"
+)
+
+// Chassis table OVN SB
+type Chassis struct {
+	UUID                string
+	Encaps              []string
+	ExternalID          map[interface{}]interface{}
+	Hostname            string
+	Name                string
+	NbCfg               int
+	TransportZones      []string
+	VtepLogicalSwitches []string
+}
+
+func (odbi *ovndb) chassisAddImp(name string, hostname string, etype string, ip string) (*OvnCommand, error) {
```

Comment:
One chassis may have N encaps. The etype and ip should be list type.

Reply:
schema shows encap type as string
Encap": {
            "columns": {
                "type": {"type": {"key": {
                           "type": "string",
                           "enum": ["set", ["geneve", "stt", "vxlan"]]}}},

Reply:
but via cli app i can set two encaps

Reply:
@vtolstov : Please look at the latest go-ovn master code. This is already addressed  and implemented :)  e.g. ovn-sbctl chassis-add ch1 stt,geneve,vxlan 1.2.3.5

Reply:
Thanks